exercises/concept/bird-watcher/.docs/hints.md

```diff
@@ -2,21 +2,21 @@
 
 ## 1. Determine the total number of birds that you counted so far
 
-- Refer to the exercise introduction for an example how to use a for loop to iterate over an array.
+- Refer to the exercise introduction for an example of how to use a for loop to iterate over an array.
 - Use a helper variable to store the total count and increase that variable as you go through the array.
 - Think about the correct initial value for that helper variable.
 - Refer back to the [array concept][concept-arrays] to recap how to retrieve values from an array.
 
 ## 2. Calculate the number of visiting birds in a specific week
 
-- This task is similar to the first one, you can copy your code as a starting point.
+- This task is similar to the first one. You can copy your code as a starting point.
```

Comment:
Another place where the split into a separate sentence might need to go on separate line.

Reply:
Yep